### PR TITLE
Fixed Powershell #1585

### PR DESCRIPTION
**Pull request by j0rt3g4**
I've clear it #1585 and also fixed notes. #1585 
Copy review @nenonix @JohanFreelancer9 is required on this one guys. Thank you in advance!

### CODE REVIEW

File: SharePoint/SharePointServer/security-for-sharepoint-server/implement-saml-based-authentication-in-sharepoint-server.md [hunk 1]
```diff
@@ -79,19 +79,20 @@ This script configures SharePoint 2013 with ADFS certificates and claim type map
...
 
 # The local file path, which points to the certificate used to sign token requests (exported from the AD FS server) $certFilePath = $ScriptDir + "\Certificates\" # Build the certificates. $adfsCertPath = $certFilePath + $adfsCertName + ".cer" $MACertName = $certFilePath + $MACertName + ".cer" $MIACertPath = $certFilePath + $MIACertName + ".cer" $RootCertPath = $certFilePath + $BCTRCertName + ".cer" 
-
 # Import certificates to the SharePoint Trusted Root Authority. # $adfsCert = $null if($adfsCert -eq $null) { Write-Host "installing " $adfsCert $adfsCert = New-Object System.Security.Cryptography.X509Certificates.X509Certificate2($adfsCertPath) 
-
 # Install the certificate that was exported from the ADFS server. New-SPTrustedRootAuthority -Name $adfsCertName -Certificate $adfsCert New-SPTrustedRootAuthority -Name $MACertName -Certificate $MACertName New-SPTrustedRootAuthority -Name $MIACertName -Certificate $MIACertPath New-SPTrustedRootAuthority -Name $BCTRCertName -Certificate $RootCertPath } 
 ```
-**Note**: You can use a different file name, but you must save the file as an ANSI-encoded text file that has the extension .ps1.
+
+ >[!NOTE]
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/security-for-sharepoint-server/implement-saml-based-authentication-in-sharepoint-server.md]:
```suggestion
 > [!NOTE]
```

File: SharePoint/SharePointServer/security-for-sharepoint-server/implement-saml-based-authentication-in-sharepoint-server.md [hunk 2]
```diff
@@ -79,19 +79,20 @@ This script configures SharePoint 2013 with ADFS certificates and claim type map
...
+ >You can use a different file name, but you must save the file as an ANSI-encoded text file that has the extension .ps1.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/security-for-sharepoint-server/implement-saml-based-authentication-in-sharepoint-server.md]:
```suggestion
 > You can use a different file name, but you must save the file as an ANSI-encoded text file that has the extension .ps1.
```

**Comment by j0rt3g4** [comment]
Hello @JohanFreelancer9  I've already added those lines please update :)

File: SharePoint/SharePointServer/security-for-sharepoint-server/implement-saml-based-authentication-in-sharepoint-server.md [hunk 3]
```diff
@@ -125,28 +129,34 @@ $ScriptDir = Split-Path -parent $MyInvocation.MyCommand.Path
...
-New-SPTrustedRootAuthority -Name $adfsCertName -Certificate $adfsCert New-SPTrustedRootAuthority -Name $MACertName -Certificate $MACertName New-SPTrustedRootAuthority -Name $MIACertName -Certificate $MIACertPath New-SPTrustedRootAuthority -Name $BCTRCertName -Certificate $RootCertPath } 
+    New-SPTrustedRootAuthority -Name $adfsCertName -Certificate $adfsCert 
+    New-SPTrustedRootAuthority -Name $MACertName -Certificate $MACertName 
+    New-SPTrustedRootAuthority -Name $MIACertName -Certificate $MIACertPath 
+    New-SPTrustedRootAuthority -Name $BCTRCertName -Certificate $RootCertPath 
+ } 
 ```
 
-**Note**: You can use a different file name, but you must save the file as an ANSI-encoded text file that has the extension .ps1.
+ >[!NOTE]
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/security-for-sharepoint-server/implement-saml-based-authentication-in-sharepoint-server.md]:
```suggestion
 > [!NOTE]
```

File: SharePoint/SharePointServer/security-for-sharepoint-server/implement-saml-based-authentication-in-sharepoint-server.md [hunk 4]
```diff
@@ -125,28 +129,34 @@ $ScriptDir = Split-Path -parent $MyInvocation.MyCommand.Path
...
+ >You can use a different file name, but you must save the file as an ANSI-encoded text file that has the extension .ps1.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/security-for-sharepoint-server/implement-saml-based-authentication-in-sharepoint-server.md]:
```suggestion
 > You can use a different file name, but you must save the file as an ANSI-encoded text file that has the extension .ps1.
```

File: SharePoint/SharePointServer/security-for-sharepoint-server/implement-saml-based-authentication-in-sharepoint-server.md [hunk 5]
```diff
@@ -190,36 +199,43 @@ $IdClaimType = http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddres
...
 **Add the partner site realm to the trusted provider**
- 
 $uri = new-object System.Uri($webAppUri) 
-$ap = Get-SPTrustedIdentityTokenIssuer $tokenIdentityProviderName $ap.ProviderRealms.Add($uri,$siteRealm) $ap.Update() 
+$ap = Get-SPTrustedIdentityTokenIssuer $tokenIdentityProviderName $ap.ProviderRealms.Add($uri,$siteRealm) 
+$ap.Update() 
 ```
 
-**Note**:  You can use a different file name, but you must save the file as an ANSI-encoded text file that has the extension .ps1.
+ >[!NOTE]
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/security-for-sharepoint-server/implement-saml-based-authentication-in-sharepoint-server.md]:
```suggestion
 > [!NOTE]
```

File: SharePoint/SharePointServer/security-for-sharepoint-server/implement-saml-based-authentication-in-sharepoint-server.md [hunk 6]
```diff
@@ -190,36 +199,43 @@ $IdClaimType = http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddres
...
+ >You can use a different file name, but you must save the file as an ANSI-encoded text file that has the extension .ps1.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/security-for-sharepoint-server/implement-saml-based-authentication-in-sharepoint-server.md]:
```suggestion
 > You can use a different file name, but you must save the file as an ANSI-encoded text file that has the extension .ps1.
```

**Comment by JohanFreelancer9** [comment]
@j0rt3g4  You were quick - sorry saw your comment too late. Thanks.

**Comment by j0rt3g4** [comment]
@JohanFreelancer9  VS magic :)

**Comment by scanum** [comment]
@MikePlumleyMSFT PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**